### PR TITLE
Added simple handlebar steering model.

### PR DESCRIPTION
**Pull request by moorepants**

### CODE REVIEW

File: examples/simple_steerer.py [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+                        frame=C,
+                        mass=mC,
+                        inertia=(IC, Co))
+radius = me.RigidBody('radius',
+                      masscenter=Do,
+                      frame=D,
+                      mass=mD,
+                      inertia=(ID, Do))
+
+steer_resistance = me.Torque(A, (-kA*q1 + cA*u2)*N.z)
```
Comment (moorepants) [inline examples/simple_steerer.py:125]:
```suggestion
steer_resistance = me.Torque(A, (-kA*q1 - cA*u2)*N.z)
```

File: examples/simple_steerer.py [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+muscle_pathway = LinearPathway(Cm, Dm)
```
Comment (moorepants) [inline examples/simple_steerer.py:126]:
```suggestion
bicep_path = LinearPathway(Cm, Dm)
```

**Comment by moorepants** [comment]
The arm steers! Simulation works with animation. Can be merged. Just needs some muscles now. :)

**Comment by moorepants** [comment]
https://github.com/brocksam/sympy-biomechanics/assets/276007/b60eb67b-667f-4bb6-9639-c57273510c6f

**Review by brocksam — APPROVED** [review]

**Pull request merged**